pkg/signal/trap.go

```diff
@@ -54,11 +53,11 @@ func Trap(cleanup func()) {
 						}
 					} else {
 						// 3 SIGTERM/INT signals received; force exit without cleanup
-						logrus.Info("Forcing docker daemon shutdown without cleanup; 3 interrupts received")
+						fmt.Println("Forcing docker daemon shutdown without cleanup; 3 interrupts received")
 					}
```

Comment:
Using stdout makes the situation worse.
Please remove logs or pass some interface to the function instead